src/pipecat/runner/run.py

```diff
@@ -166,6 +166,7 @@ def _create_server_app(
     host: str = "localhost",
     proxy: str,
     esp32_mode: bool = False,
+    whatsapp_mode: bool = False,
```

Comment:
nit: I wonder if something like `whatsapp_enabled` would be better in this case, since we’re basically using it to enable or disable the WhatsApp routes.

Reply:
makes sense. it's more descriptive so I'm ok with changing it to `enabled`.